sympy/integrals/integrals.py

```diff
@@ -758,9 +763,19 @@ def _eval_integral(self, f, x, meijerg=None, risch=None, manual=None,
             else:
                 if i:
                     # There was a nonelementary integral. Try integrating it.
-                    return result + i.doit(risch=False)
+                    a="ans"
```

Comment:
Ummm @AnishShah . Not really. I had created this new branch right from the start. Just switched to it, before committing and pushing this PR. :smile: